feat: interactive debug in tests (#455)
ApeWorX/ape: docs/userguides/projects.md

```bash
ape plugins add hardhat
```

### Test Interactive

> Use ape test -I to open the interactive mode at the point of exception. This allows the user to inspect the point of failure with all the relavant namespaces.

Reviewer: ```suggestion
Use ape test -I to open the interactive mode at the point of exception. This allows the user to inspect the point of failure in your tests.
```
